mapzen-places-api/src/main/java/com/mapzen/places/api/internal/AutocompletePendingResult.java

```diff
@@ -32,6 +34,7 @@
   private final String query;
   private final LatLngBounds bounds;
   private final AutocompleteFilter filter;
+  private final DialogDisplayer dialogDisplayer = new SettingsDialogDisplayer();
```

Comment:
It seems like an unfortunate side effect of the Lost `SettingsApi` implementation that the `Places` API would need to know about a `DialogDisplayer`. We should rethink how to better encapsulate this behavior moving forward.

In the meantime do we need to return a real `SettingsDialogDisplayer` in this class or can we just make this param `null` when creating a new `Status`?